DARPA-ASKEM/terarium
# feat: create subset data operator

# Description

- Created a subset operator that accepts an esgf dataset
- Latitude and longitude coordinates can be entered as integers
- Then click Run to generate the subset
- Can save the subset as a new dataset
- Preview images still have to be worked on in the backend 
- Spatial and time skipping isn't passed to the endpoint yet

### Issues
- ~After the first run I get an error - it seems like a new subset can't be generated after that (it worked sometimes before though)~ resolved with poller

### Testing notes
- Search "historical" and switch the dropdown source to ESGF in the explorer and add one of those datasets to a project
- Make sure you use `yarn dev` since staging connects to the climate data service 


https://github.com/DARPA-ASKEM/terarium/assets/28237258/2de83641-526a-4c71-868e-14b529923fe5
